pkg/asset/installconfig/ssh.go

```diff
@@ -60,56 +60,21 @@ func (a *sshPublicKey) Generate(map[asset.Asset]*asset.State) (state *asset.Stat
 		sort.Strings(paths)
 	}
 
-	promptLines := []string{"SSH Public Key:"}
-	if len(paths) == 0 {
-		promptLines = append(
-			promptLines,
-			"Enter an empty string or your public key (e.g. 'ssh-rsa AAAA...')",
-		)
-	} else {
-		promptLines = append(
-			promptLines,
-			"Enter an empty string, your public key (e.g. 'ssh-rsa AAAA...'), or one of the following numbers:",
-		)
-		for i, path := range paths {
-			promptLines = append(
-				promptLines,
-				fmt.Sprintf("%d: %s", i+1, path),
-			)
-		}
-	}
-	prompt := strings.Join(promptLines, "\n")
-
-	var input string
-	for {
-		fmt.Println(prompt)
-		input, err = a.inputReader.ReadString('\n')
-		if err != nil && err != io.EOF {
-			fmt.Println("Could not understand response. Please retry.")
-			continue
-		}
-		if input != "" && input[len(input)-1] == '\n' {
-			input = input[:len(input)-1]
-		}
-		var i int
-		n, err := fmt.Sscanf(input, "%d", &i)
-		if n == len(input) && err == nil && i > 0 && i <= len(paths) {
-			path := paths[i-1]
-			input = pubKeys[path]
-		} else {
-			err = validate.OpenSSHPublicKey(input)
-			if err != nil {
-				fmt.Println(err)
-				continue
-			}
-		}
-		break
+	var path string
+	survey.AskOne(&survey.Select{
+		Message: "SSH Public Key",
+		Help:    "The SSH key used to access all nodes within the cluster. This is optional.",
+		Options: append([]string{none}, paths...),
```

Comment:
I think you also want to continue to support "paste in your SSH pubkey" (vs. just reading it from a file).

Reply:
We can just tell the user to edit their install config if they want to do that. Remember, this interactive UI is for the _basic_ use case - nothing fancy.